style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  background-color: #fff;
+}
+
+.header {
+  position: absolute;
+  width: 375px;
+  left: calc(50% - 375px / 2);
+  display: flex;
+  justify-content: space-around;
+  border: 1px solid gray;
+  background-color: azure;
+}
```

Comment:
- [ ] Here you can remove the `position: absolute;` and the ` background-color`.

Reply:
@ridwanediallo 
That will mean I also need to remove the `left: calc(50% - 375px / 2);` as it can only work with `position: absolute;` in this case. 
Now is it necessary to include `width: 375px`?